torch/distributed/fsdp/common_utils.py

```diff
@@ -0,0 +1,35 @@
+"""
+Public common utilities for FSDP.
+"""
+
+from enum import auto, Enum
+
+
+class BackwardPrefetch(Enum):
+    """
```

Comment:
does this suppose to show in our API doc?

Reply:
Yes. I will make sure the docs are rendering correctly before landing.